lib/nfg/capistrano/config.rb

```diff
@@ -0,0 +1,28 @@
+module Nfg
+  module Capistrano
+    module Config
+      def self.circleci
+        @circleci_config ||=
+          ActiveSupport::HashWithIndifferentAccess.new(
+            YAML::load_file('.circleci/config.yml', aliases: true)
```

Comment:
File this is loading: https://github.com/network-for-good/Givecorps-site/blob/develop/.circleci/config.yml